SyphonNameboundClient.m

```diff
@@ -216,7 +216,8 @@ + (NSDictionary *)serverInfoFromNotification:(NSNotification *)notification
     // This deals with a change in notifications from Syphon.
     // Once all projects using SyphonNameboundClient are updated to newer Syphon.framework
     // we won't need this method and can always use notification.userInfo.
-    if ([notification.object isKindOfClass:[SyphonServerDirectory class]])
+    //if ([notification.object isMemberOfClass:[SyphonServerDirectory class]])
+	if ([notification.userInfo objectForKey:SyphonServerDescriptionUUIDKey] != nil)
```

Comment:
Does it make sense to use NSClassFromString and pass in the MACRO value for the SyphonServerDirectory compatibility alias? Im remote right now so cant 100% check (shitty service and im on vacation <3 )

Just thinking aloud! Thanks for the PR @robtherich !!!!